conf/modules.config

```diff
@@ -44,6 +44,55 @@ process {
         ext.prefix = { "${meta.id}.fa" }
     }
 
+    withName: MMSEQS_DATABASES {
+        publishDir = [
+            path: { "${params.outdir}/databases/" },
+            mode: params.publish_dir_mode,
+            enabled: params.save_databases,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+        ext.args = [
+            params.taxonomy_mmseqs_databases_savetmp ? "" : "--remove-tmp-files" ,
+        ].join(' ').trim()
+    }
+
+    withName: MMSEQS_CREATEDB {
+        publishDir = [
+            path: { "${params.outdir}/taxonomy/mmseqs_createdb/" },
```

Comment:
`taxonomy` here is referring to an actual taxonomy set of files (e.g. nodes.dmp, names.dmp), or the output of taxonomic classification? If the latter I think it would make more sense to call it `taxonomic_classification/`. But tpersonal opinion!